src/index_emails.py

```diff
@@ -39,9 +36,8 @@ def strip_html_css_js(msg):
 
 def delete_index():
     try:
-        url = "%s/%s?refresh=true" % (tornado.options.options.es_url, tornado.options.options.index_name)
+        url = "%s/%s" % (tornado.options.options.es_url, tornado.options.options.index_name)
```

Comment:
Question: why did you remove the refresh=true here and in body below? (it's been a while since i've worked with ES and I've not kept up with newer versions)

Reply:
I'm using ES 7.7. When setting that parameter I'm getting a HTTP status code 400 with a root cause 'illegal_argument_exception' -> 'request [...] contains unrecognized parameter'.
The index API docs at https://www.elastic.co/guide/en/elasticsearch/reference/7.7/indices-delete-index.html also don't mention any refresh parameter.

On the other hand, the documents API seems to support such an parameter: https://www.elastic.co/guide/en/elasticsearch/reference/7.7/docs-refresh.html

Maybe this was changed in later ES version?

Reply:
Very possible. As I said, not really up to date with my ES knowledge.
Thanks for linking the docs, this should suffice for now as documentation.